connectors/sources/azure_blob_storage.py

```diff
@@ -108,6 +109,27 @@ def get_default_configuration(cls):
             },
         }
 
+    @classmethod
```

Comment:
Here we probably also want to use the class-level fields?

Reply:
I forgot to remove this 😅